container/Exchange.js

```diff
@@ -32,6 +32,10 @@ export default () => {
   const [translation, setTranslation] = useState(undefined)
 
   useEffect(() => {
+    if (search === undefined) {
```

Comment:
Does this also work for `localhost:3000/?search=` or would `search` be an empty string in that case? What do you think of doing this instead: 
```suggestion
    if (!search) {
```

I think we catch more cases that way.

Reply:
Yes good point!

Reply:
This looks like it was resolved by mistake